### PR TITLE
Particle Integration

### PR DESCRIPTION
**Pull request by Noobulater**
Creates a particle system as a webaverse app

Allows us to add more effects using editor 

>temp link https://demo.mythicarchive.net?fx=1

### CODE REVIEW

File: world.js [hunk 1]
```diff
@@ -18,6 +18,20 @@ import {worldMapName, appsMapName, playersMapName} from './constants.js';
...
+    await metaverseModules.waitForLoad();
+    const {modules} = metaversefileApi.useDefaultModules();
+    const m = modules['specialEffect'];
+    await effectApp.addModule(m);
+
+    effectApp.position.set(-2,2,-2)
+    effectApp.updateMatrixWorld();
+  })();
+
+  scene.add(effectApp);
```
Comment (avaer) [inline world.js:33]: We generally should not add things to the scene unless we are an app. And this is not an app, this is the world bootstrap.

File: world.js [hunk 2]
```diff
@@ -18,6 +18,20 @@ import {worldMapName, appsMapName, playersMapName} from './constants.js';
 import {playersManager} from './players-manager.js';
 import * as metaverseModules from './metaverse-modules.js';
 
+setTimeout(()=>{
```
Comment (avaer) [inline world.js:21]: We shouldn't run this app by default right? We can preload the module but then not instantiate it. I imagine the effects system (accessed via totum APIs) would intantiate instances of the effects app as needed and coordinate.

File: world.js (hunk 1 above)
Comment (Noobulater) [reply world.js:33]: Yea shouldn't be run by default. I'll adjust the PR, I was eager to show something working, but these changes are inappropriate for master

File: world.js (hunk 2 above)
Comment (Noobulater) [reply world.js:21]: Correct, its just bootstrap.

**Comment by avaer** [comment]
@Noobulater is not available to finish the work.

**Pull request closed without merging**